### ISSUE TITLE
Suggestion: Include EMB file type for read and write

### ISSUE THREAD

**Issue by JohnMPC** [Embroidermodder/Embroidermodder#277]
Hi, I am working on an embroidery order management project. I'm hoping that once your development is complete in python then i will be able to use your software within mine.  I'm not sure if you have heard of EMBtrak. They are a US based software company that have a bit of software that allows the user to input orders and combine it with the embroidery file and colour information with in the main module. Then produce production worksheets with all the required information. This product can be liscenced for a crazy amount of money. But i'm hoping to produce something similar and take it to the next level.  
In the UK Wilcom is probably the most used Digitising software. The standard file type is EMB, this will save multiple colorways for the same design in 1 file. It would be a great help if you would considering adding this to the supported file types for embroidermodder2. 

Thanks for your efforts and can't wait to see EM2!!

**Comment by robin-swift** [comment]
Hi @JohnMPC,

as it stands, if you're looking to interface basic read/write of a specific file format (even in Python) then that would be a feature request for [libembroidery](https://github.com/Embroidermodder/libembroidery). So I can create "support emb format" as an [issue there](https://github.com/Embroidermodder/libembroidery/issues/199). As far as I can tell there's no reason why we wouldn't build that when we get round to it.

As for

> i'm hoping to produce something similar and take it to the next level.

If you post your working draft as a GitHub repository then I can help you use our under-documented and incomplete API. In fact doing so would help me improve the API for future use.

Since this is a hobbyist's project if you want to see a specific feature prioritised then consider donating to our [Open Collective page](https://opencollective.com/embroidermodder) a fraction of the money you would spend on licensing someone else's software. In exchange, we can spend more of our time on the project in general and your idea in specific.

Looking forward to working with you either way.

Cheers,

Robin
(The Embroidermodder Team)

**Comment by JohnMPC** [comment]
Hi @robin-swift. Thank you for taking the time to respond to me. 
My skills and project are along way behind yours. When i have the majority of the app working i will be happy to share. 
The main use i am looking for from embroidermodder will be to push and pull colourways. These will be saved in the order database next to the relavent garment.

**Comment by robin-swift** [comment]
> The main use i am looking for from embroidermodder will be to push and pull colourways. These will be saved in the order database next to the relavent garment.

This is completely new to me, could you show me a screenshot of this in some other editor or something? Maybe as a new issue because this is separate from .emb file format support.

Cheers, Robin

**Comment by JohnMPC** [comment]
I'll send you an email through the website so we have direct contact details.
